### DISCUSSION TITLE
Need to be able to "revert/undo" custom sort/column adds by a user in control panel

### DISCUSSION THREAD

**Discussion by matthewstick** [craftcms/cms#14681] [Ideas] [answered]
Craft added a feature that lets users customize their control panel element views/element index pages.  I'm referring to all “views” in the control panels: singles, channels, structures, etc.  

Users can custom sort/column adds by a user. And it also saves this.  So when a user returns to the control panel, it remembers what they did.

We need a way to enforce the view and make it NOT customizable.  Or even better, that they can customize it during a session, but when they log back in it shows the "default"

Like say I have 5 users.  Now say each user sorts differently. Craft currently "remembers" how each user sorted, even when they log in and log back out.  It always shows.a custom sort and column set up per user.  It never “shows the default” column sort.

Likewise, users can customize their own columns.  It shows the user's custom columns, even if they log back in and out.

What I want is for there to “reset” to the default view.

There its no way to “undo” the custom sort/column adds by a user.   Not that I know of at least.  We have a big team of users and always need specific views to “start at the default” not “start at where that particular user last left it”.  It is getting really messy.

For example, image a element index is sorting my "job due date" - with most urgent stuff at the top.  then imaging some user sorts different and forgets they sort it because they aren't savvy.  They will miss the fact that somethign is due.  Now imagine 50 people all accessing that view, and everyone has a different version.  It's a mess!

Basically, I want the reverse of this:
https://github.com/craftcms/cms/discussions/11355#discussioncomment-2869454

Thanks!

**Comment by brandonkelly** [comment, marked as answer]
> There its no way to “undo” the custom sort/column adds by a user.

That’s what the “Use defaults” button at the bottom of the View menu is for.

**Comment by matthewstick** [comment]
That "use defaults" feature is kind of buried though.  I've been a power craft cms developer since carft 2 and never saw that before :) Let alone a kind of user who can barely work the computer.

We'd like the ability to enforce it to not be customizable.  Or even better, that they can customize it during a session, but when they log back in it shows the "default."  Like a "reset views to default on user log" in option for each element index.

**Reply by brandonkelly** [reply]
It’s right in the same HUD as the rest of the view controls so I’m not sure I’d call it “buried”. But I reopened the discussion since you want something automated.
